### DISCUSSION TITLE
I want to use init_bulk to create alloys with different concentrations, which keywords should I use in param.josn to use this function？def create_random_alloys(fin, alloy_dist, ifmt='vasp', ofmt='vasp')

### DISCUSSION THREAD

**Discussion by Saberve** [deepmodeling/dpgen#756] [Q&A]
I want to use init_bulk to create alloys with different concentrations, which keywords should I use in param.josn to use this function？
def create_random_alloys(fin, alloy_dist, ifmt='vasp', ofmt='vasp'):
![image](https://user-images.githubusercontent.com/53855907/173547380-c4224de3-30b5-4a1c-b759-ff5828850435.png)
![image](https://user-images.githubusercontent.com/53855907/173547474-3e81721f-6a08-44de-84e1-c64719f80294.png)

**Comment by HuangJiameng** [comment]
Maybe you can use POSCAR of alloys with different concentrations to do init_bulk, which can be found on materialsproject.org.
